### PR TITLE
stat/distuv: fix CDF and score functions for Weibull distribution

### PR DESCRIPTION
**Pull request by romanwerpachowski**
Also completes test coverage for Weibull.

### CODE REVIEW

File: stat/distuv/weibull.go [hunk 1]
```diff
@@ -46,14 +45,6 @@ func (w Weibull) gammaIPow(i, pow float64) float64 {
...
 }
 
-// LogCDF computes the value of the log of the cumulative density function at x.
-func (w Weibull) LogCDF(x float64) complex128 {
-	if x < 0 {
-		return 0
-	}
-	return cmplx.Log(-1) + complex(-math.Pow(x/w.Lambda, w.K), 0)
-}
-
```
Comment (kortschak) [inline stat/distuv/weibull.go:56]: Is there a way to retain the `LogCDF` without wandering into complex territory?

Comment (romanwerpachowski) [reply stat/distuv/weibull.go:56]: There is, but does anyone need it? AFAIK it's the only distribution in `stat/distuv` which has this function.

Comment (kortschak) [reply stat/distuv/weibull.go:56]: No, it's not.

```
$ go doc -all gonum.org/v1/gonum/stat/distuv | grep func.*LogProb
func (b Bernoulli) LogProb(x float64) float64
func (b Beta) LogProb(x float64) float64
func (b Binomial) LogProb(x float64) float64
func (c Categorical) LogProb(x float64) float64
func (c ChiSquared) LogProb(x float64) float64
func (e Exponential) LogProb(x float64) float64
func (f F) LogProb(x float64) float64
func (g Gamma) LogProb(x float64) float64
func (g GumbelRight) LogProb(x float64) float64
func (g InverseGamma) LogProb(x float64) float64
func (l Laplace) LogProb(x float64) float64
func (l LogNormal) LogProb(x float64) float64
func (n Normal) LogProb(x float64) float64
func (p Pareto) LogProb(x float64) float64
func (p Poisson) LogProb(x float64) float64
func (s StudentsT) LogProb(x float64) float64
func (t Triangle) LogProb(x float64) float64
func (u Uniform) LogProb(x float64) float64
func (w Weibull) LogProb(x float64) float64
```

I think there's only one that doesn't have it, `AlphaStable`.

Comment (romanwerpachowski) [reply stat/distuv/weibull.go:56]: You searched for `LogProb`, but I deleted `LogCDF`.

Comment (kortschak) [reply stat/distuv/weibull.go:56]: Eeep. Yes. Apologies. Delete away.

**Pull request merged**